### DISCUSSION TITLE
Show ALL blog posts

### DISCUSSION THREAD

**Discussion by reeceatkinson** [facebook/docusaurus#7602] [Q&A] [answered]
I would like the option to get a page where it shows ALL the blog posts of a site not just recent posts as shown below. Is this possible or built in?
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/78908925/173185019-09f3b9e5-2406-4100-b9e0-efd014ccae68.png">

**Comment by Josh-Cena** [comment, marked as answer]
Please read through the docs. https://docusaurus.io/docs/blog#blog-sidebar

**Reply by Tautorn** [reply]
Thanks a lot!

**Reply by electrovir** [reply]
What about a dedicated page that shows all blog posts? How is a user intended to find all blog posts if the blog only shows recents in the side bar? I want both behaviors.

**Reply by electrovir** [reply]
Answer to my own question: this isn't mentioned anywhere in the docs but it's the `/archive` page.

**Reply by Tautorn** [reply]
@electrovir there's a way to show the all blog post:
```
const recentPosts = require("../../.docusaurus/docusaurus-plugin-content-blog/default/blog-post-list-prop-default.json");
```

But that option is limited, the json returned dont has all items about the post.

Iteration example:

```jsx
{recentPosts.items.slice(0, 5).map((item, index) => (
            <div >
              <div style={{fontSize:'1.25rem', padding:'10px', margin:'2px'}} key={index}>
              <img src="/img/svg/developer_activity.svg" height="35px" />
                <a href={`${item.permalink}`}>{item.title}</a>{" "}
              </div>
            </div>
          ))}
```

But I don't use this option. I did create a json with my last posts and show it on the page. The problem with that approach is that it is manual.
